geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/mapserver/mapserver.map.tmpl

```diff
@@ -22,7 +22,7 @@ MAP
     EXTENT {{cookiecutter.extent_mapserver}} ## should be defined for better performance
     UNITS METERS
 
-    # RESOLUTION and DEFRESOLUTION default to 96. If you
+    # RESOLUTION and DEFRESOLUTION default is 96. If you
```

Comment:
here, the "default to" was correct, because of the plural and therefore "default" can be read as a verb
So:
RESOLUTION and DEFRESOLUTION default to 96
or
RESOLUTION and DEFRESOLUTION have as default value 96
or
For RESOLUTION and DEFRESOLUTION, the default is